Packages/vcs/Lib/vcsvtk/pipeline.py

```diff
@@ -9,15 +10,28 @@ class Pipeline(object):
     VTK plot command. Refer to the method documentation for details.
     """
 
-    def __init__(self, context_):
+    def __init__(self, graphics_method, context_):
         """Initialize the pipeline object.
 
+        _gm is a vcs graphics method
         _context is a weakref of the VTKVCSBackend object that created this
             Pipeline.
         """
         self._context = weakref.ref(context_)
+        self._gm = graphics_method
 
     # For now, we'll just throw everything at plot. This might need to be
     # broken up into set_data, set_template, etc methods...
     def plot(self, data1, data2, template, graphics_method, grid, transform):
         raise NotImplementedError("Missing override.")
+
+    def getcolormap(self):
```

Comment:
shouldn't it be called getColorMap(self) so as to be consistent with rest of the naming convention?